packages/handlebars-templates/src/index.ts

```diff
@@ -290,6 +290,30 @@ export function registerStandardHelpers<O>(hbs: typeof Handlebars, { utils }: Co
 		}
 	})
 	
+	hbs.registerHelper('ifdef', function(this: object, a: unknown, options: Handlebars.HelperOptions) {
+		if (!options) {
+			throw new Error('ifdef helper must be called with two arguments')
```

Comment:
This should be "one argument" rather than "two arguments". The helper this was copied from (`ifeq`) requires two arguments, `a` and `b`, but this one just wants one, `a`.